pkg/virtual/workspaces/builder/build.go

```diff
@@ -109,9 +111,21 @@ func BuildVirtualWorkspace(rootPathPrefix string, workspaces workspaceinformer.W
 						return nil, err
 					}
 
+					var once sync.Once
+					var workspacesRest rest.Storage
+					var kubeconfigSubresourceRest rest.Storage
+					buildRestForWorkspaceResourceAndSubResources := func() {
+						workspacesRest, kubeconfigSubresourceRest = virtualworkspacesregistry.NewREST(kcpClient.TenancyV1alpha1(), kubeClient, crbInformer, reviewerProvider, workspaceAuthorizationCache)
```

Comment:
why the func? Why not in-place?

Reply:
Fixed in commit 2c97666d51d66ccc901f7d6f92992f5ba1412a5d